_data/changelogs/component-tooltip.yml

```diff
@@ -5,6 +5,13 @@ type: component
 changelogURL:
 
 items:
+  - date: NNNN-NN-NN
+    summary: "Restored the `opacity: 0` style rule to the tooltip's initial state."
```

Comment:
Should this be in quotes? Looks like we don't normally quote our summaries

Reply:
@amyleadem are the quotes required because of the backticks?

Reply:
Yes, I get a syntax error otherwise.

Reply:
For clarity, I believe it is the colon within the backticks that made the quotes necessary